src/components/Common/Footer/Footer.module.scss

```
@@ -0,0 +1,69 @@
+@import 'styles/global.scss';
+
+footer {
+  background-color: #f2f2f2;
+  font-size: 14px;
+  .footerWrapper {
+    max-width: calc(100% - 320px);
+    margin: 20px auto;
+  }
+
+  .footerCase {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid #c4c4c4;
+    padding: 22px 0;
+    color: black;
+    line-height: 17.53px;
+  }
+
+  .footerLink {
+    display: flex;
+    align-items: center;
+
+    li:nth-child(3) {
+      font-weight: 700;
+    }
+
+    li {
+      &:after {
+        content: '|';
+        margin: 0 14px;
+        font-size: 14px;
+        line-height: 17.53px;
+      }
+    }
+
+    .lastText {
+      &:after {
+        content: '';
+      }
+    }
+  }
```

마지막 요소에서만 가상요소를 없애고 싶다면,
```suggestion
    li {
      &:not(:last-child)::after {
        content: '|';
        margin: 0 14px;
        font-size: 14px;
        line-height: 17.53px;
      }
    }
  }
```
이렇게도 사용가능할것같아요